src/Options.php

```diff
@@ -729,6 +747,7 @@ private function configureOptions(OptionsResolver $resolver): void
             'capture_silenced_errors' => false,
             'max_request_body_size' => 'medium',
             'class_serializers' => [],
+            'fatal_error_handler_reserved_memory_amount' => null,
```

Comment:
We can set this value to `ErrorHandler::DEFAULT_RESERVED_MEMORY_SIZE`. This way, we can revert the change to the allowed types and in the end we can remove the fallback to the constant's value in the getter